frontend/src/ProductionHelper.tsx

```diff
@@ -62,17 +68,28 @@ export default function ProductionHelper() {
                 handleBudgetFilterChange
             )}
 
-            {Object.values(Unit).map((unit, i) => (
-                <UnitCounter
-                    key={i}
-                    unit={unit}
-                    unitCost={getUnitCost(unit)}
-                    unitCount={unitCounts.get(unit) || 0}
-                    imgPath={getImgPath(unit)}
-                    onChange={handleUnitCountChange}
-                />
-            ))}
-        </div>
+            <Stack>
+                {Object.values(Unit).map((unit, i) => (
+                    <UnitCounter
+                        key={i}
+                        unit={unit}
+                        unitCost={getUnitCost(unit)}
+                        unitCount={unitCounts.get(unit) || 0}
+                        imgPath={getImgPath(unit)}
+                        onChange={handleUnitCountChange}
+                    />
+                ))}
+            </Stack>
+
+            <Box>
+                <Button
+                    color={"danger"}
+                    onClick={() => handleClearUnitCounts()}
```

Comment:
Nit: `onClick={handleClearUnitCounts}`. Nbd if you'd rather just merge though!

Reply:
What, you don't think I should've went the other way and gone with `() => (() => handleClearUnitCounts())()` ??

Reply:
😂